### ISSUE TITLE
Missing an Ingress Kubernetes User Guide

### ISSUE THREAD

**Issue by ybizeul** [containous/traefik#5855]
### Do you want to request a *feature* or report a *bug*?

Feature

### What did you expect to see?

The "User Guides" section of the official documentation has only one section for CRDs in "Kubernetes and Let's Encrypt".

If one was not interested in using CRD, it would be extremely hard to figure out how to configure RBAC for example. There should be a dedicated section for simple Ingress / Kubernetes configuration.

The documentation has many, many shortcomings, very confusing in many places, but that's one of the area where a simple fix is possible.

**Comment by RakeshNagarajan** [comment]
Same issue, No proper documentation, moving to nginx ingress

**Comment by zakkg3** [comment]
https://docs.traefik.io/migration/v1-to-v2/#strip-and-rewrite-path-prefixes
its broken.

```
apiVersion: traefik.containo.us/v1alpha1
kind: IngressRoute
metadata:
  name: http-redirect-ingressRoute
```

this name could not be uppercase.
after fixing this, another error:

```
reflector.go:125] pkg/mod/k8s.io/client-go@v0.0.0-20190718183610-8e956561bbf5/tools/cache/reflector.go:98: Failed to list *v1alpha1.IngressRoute: v1alpha1.IngressRouteList.Items: []v1alpha1.IngressRoute: v1alpha1.IngressRoute.v1alpha1.IngressRoute.Spec: v1alpha1.IngressRouteSpec.Routes: []v1alpha1.Route: v1alpha1.Route.Services: []v1alpha1.Service: v1alpha1.Service.Port: readUint32: unexpected character: �, error found in #1 byte of ...|flower|..., bigger context ...|flower|...
```

**Comment by traefiker** [comment]
Closed by #6440.
